projects/ccf-rui/src/styles.scss

```diff
@@ -50,22 +50,18 @@ mat-icon {
 
 // Reference: https://css-tricks.com/custom-scrollbars-in-webkit/
 ::-webkit-scrollbar {
```

Comment:
Not sure if you need to change all of this? Have you confirmed with Leonard and/or Bruce? Personally I think it looks quite ugly with these settings.

Reply:
Although there are no examples of sidebars scroll in the spec, page 18 shows a scrollbar in the info modal that looks like the ones produced by the previous settings.

Reply:
Update: Changes are according to spec.